README.md

```diff
@@ -1263,6 +1264,24 @@ If you're already using Python 3.7, switch the `language_version` accordingly. F
 `stable` is a branch that tracks the latest release on PyPI. If you'd rather run on
 master, this is also an option.
 
+## GitHub Actions
+
```

Comment:
It should be clear that this just lists all the files that are to be formatted, just so that there is an explcit contrast between `black`'s default behaviour.

Reply:
It doesn't just list them; CI will fail to pass or linting is wrong.

This should be obvious; CI obviously won't modify a commit you've already committed  and pushed.

Reply:
Well it's not about modifying, it's just that the output is not directly available to the user, one has to jump through atleast three clicks to see the output.

Also, they could be expecting how other linting tools work with GH actions using https://github.com/reviewdog/reviewdog, which is just not possible with _Black_.

Reply:
The intent behind this action is to allow other projects to very easily run `black`as part of their CI. This is mostly to ease reviewing PRs (eg: has this person black'd their code?), or to avoid slipups. Generally, seeing "oh, you need to run black on this" is enough.

I've added `--diff` to this Action, since that shows the output too, which is definitely an improvement.

I was unfamiliar with reviewdog, and it looks really cool. I does, however, required quite a bit more setup than this which  "just runs black".

Reply:
(FWIW, running `black` with `reviewdog` should work passing just a diff.